content/en/docs/tasks/tools/install-kubectl.md

```diff
@@ -100,24 +100,22 @@ If you are on macOS and using [Macports](https://macports.org/) package manager,
 If you are on Windows and using [Powershell Gallery](https://www.powershellgallery.com/) package manager, you can install and update kubectl with Powershell.
 
 To install:
-* Run the installation commands (making sure to specify a DownloadLocation):
+* Run the installation commands (making sure to specify a `DownloadLocation`):
```

Comment:
Please update the * to instead follow the structure shown in the ## Install with Chocolatey on Windows section below (using numbers instead).  The * is not rendering properly